### DISCUSSION TITLE
Ideas

### DISCUSSION THREAD

**Discussion by amerkoleci** [amerkoleci/Vortice.Windows#259] [General]
Hi all,
Would  you like to have access to DirectXTK12 port based on vortice.windows?

Is something you would consider using? Sponsor to have support for?

Same applies to DirectXTex, DirectXMesh.

Suggestions?

**Comment by Axiverse** [comment]
Maybe not a lot of demand for DirectML yet, but the same would go for DirectMLX

**Reply by Axiverse** [reply]
How are you thinking of structuring these helper projects? Separate repos or more projects on this repo? I have a fair amount of DMLX ported in my own project, but currently interweaved with my other libs.

**Reply by amerkoleci** [reply]
I plan to have new separated repo, not that much time at the moment tho :)

**Comment by amerkoleci** [comment]
@paulbartrum what do you think about having something like DirectXTK using Vortice.Windows?

**Reply by paulbartrum** [reply]
I've used this project (DirectXTK) quite a lot, for inspiration as to how to build a memory (sub-)allocator, DDS loading, how to play audio, and a few other things. But I'm not sure a direct port is that useful. It's more like sample code than a library, at least to me. That's not to say that sample code isn't useful! But I doubt many commercial games use DirectXTK as a library, and I guess the number of people who would want this ported to .NET/Vortice is pretty small.

**Reply by dwulive** [reply]
Many games use it while prototyping* and/or in their content pipelines and/or when there are unfortunate couplings between dependencies*.
It would be nice to have.

*  which is _always_ ;)
